ironfish-cli/src/command.ts

```diff
@@ -181,6 +181,7 @@ export abstract class IronfishCommand extends Command {
       configName: typeof configFlag === 'string' ? configFlag : undefined,
       dataDir: typeof dataDirFlag === 'string' ? dataDirFlag : undefined,
       logger: this.logger,
+      executionContext: 'CLI',
```

Comment:
We already have this, it exists here. https://github.com/iron-fish/ironfish/blob/master/ironfish/src/platform.ts#L33

Reply:
TY! nice call